ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -7888,6 +7888,44 @@ def test_torch_instance_div_(
     )
 
 
+# divide
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch_frontend.div",
+    method_name="divide",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+    ),
```

Comment:
Please generate `rounding_mode` in the helper.

Reply:
@xoiga123 Sure, Thanks for the feedback. I will update you as soon as possible.